packages/gatsby-theme-docs/gatsby-browser.js

```diff
@@ -13,9 +13,7 @@ import { SWRConfig } from 'swr';
 import { docsCache } from './utils/create-emotion-cache';
 import { PortalsContainer } from '@commercetools-docs/ui-kit';
 import * as Sentry from '@sentry/browser';
-import '@fontsource/roboto/latin-400.css';
-import '@fontsource/roboto/latin-500.css';
-import '@fontsource/roboto/latin-700.css';
+import '@fontsource-variable/inter';
```

Comment:
@nkuehn I know that the font feature settings still don't have an effect on the string literals. I was also looking for a fix. I tried out the `fontsource-variable` package now, but the effect is still the same.